backend/person-service/src/main/kotlin/com/nosql/personservice/converter/PersonDtoToPersonEntityConverter.kt

```diff
@@ -0,0 +1,34 @@
+package com.nosql.personservice.converter
+
+import com.nosql.personservice.dto.PersonDto
+import com.nosql.personservice.entity.PersonEntity
+import org.bson.types.ObjectId
+import org.springframework.core.convert.converter.Converter
+import org.springframework.stereotype.Component
+
+@Component
+class PersonDtoToPersonEntityConverter(
+    private val jobTimeDtoToJobTimeEntityConverter: JobTimeDtoToJobTimeEntityConverter,
+    private val confidentialDtoToConfidentialEntityConverter: ConfidentialDtoToConfidentialEntityConverter,
+    private val contactsDtoToContactsEntityConverter: ContactsDtoToContactsEntityConverter,
```

Comment:
maybe use the conversion service with lazy initialization here?

Reply:
Fixed, but we can have troubles)))))